tools/accuracy_checker/requirements-extra.in

```diff
@@ -30,7 +30,7 @@ pyclipper>=1.2.1
 # brain tumor segmentation
 nibabel>=3.2.1
 # nncf's upperbound: https://github.com/openvinotoolkit/nncf/blob/a8af39627647ddbb89ef33cf42ff42e9c048b981/setup.py#L106
```

Comment:
We've probably missed it in the last PR. Please remove the comment because you've removed the upper bound.

I checked nncf's develop branch and they still have that requirement. Setting `scipy>=1.11.1` is going to cause conflicts.